egs/swc/s5b/local/swc_ihm_data_prep.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash 
+
+# Copyright 2016, The University of Sheffield (Author: Yulan Liu)
+# SWC IHM data preparation (train, dev and eval), as well as scoring pareparation.
+# Apache 2.0
+
+# To be executed from one directory above this script.
+
+. path.sh
+
+export LC_ALL=C
+
+#check existing directories
+if [ $# != 2 ]; then
+  echo "Usage: swc_ihm_data_prep.sh  <path to SWC data folder>  <mode>"
+  exit 1;
+fi
+
+SWC_DIR=$1      # SWC1, SWC2, SWC3 should be in subfolders under this directory
+MODE=$2
+
+dir=data/ihm/$MODE/     # Output dir
+mkdir -p $dir
+
+
+# ---------------------------------------------------------------------------
+# Explanation of the "MODE":
+# 
+#       SA: stand-alone
+#       AD: adaptation
+#
+# SWC1, SWC2 and SWC3 all have three strips: A, B, C. 
```

Comment:
What is the A,B,C? Are those the subsets of SWC* ? Also, it seems that there are many 'data/*' folders. Would it be possible to merge them into the standard 'train' 'dev' 'eval' dirs? The subsets can be defined in the 'STM' by using the 'subset identifiers enclosed in angle brackets' ('sclite' can print the WER for all the subsets in the *.lur file). ? Or is there a practical reason to have them separately (like different conditions => different processing)?